blinky.md

```diff
@@ -18,6 +18,8 @@ In your command line, make a folder for your Tessel code, then initialize a Tess
 
 `t2 init`
 
+Have a look at [the cli docs](https://tessel.gitbooks.io/t2-docs/content/API/CLI.html#starting-projects) to explore `t2 init` command.
```

Comment:
One of the most important design factors in Tessel's first run is that getting started doesn't take very long. This means the design and instructions are intentionally very streamlined and uncluttered– making https://github.com/tessel/t2-start/issues/158 a design challenge! We need to make sure that the information is available, but does not feel like an extra step users have to take before moving on.

With this current suggestion, going to look at the docs is a part of the main instructions– but it's not actually necessary that they do it before moving forward. How can we make the distinction clear?

In some places, we use italic text to provide further information:
![screen shot 2017-07-24 at 1 53 49 pm](https://user-images.githubusercontent.com/454690/28544309-930af5fa-7077-11e7-87c5-0bb9f1c900c5.png)

In other places, we call out additional non-necessary exercises with bold bonus annotation:
![screen shot 2017-07-24 at 1 54 37 pm](https://user-images.githubusercontent.com/454690/28544350-a931ad4c-7077-11e7-94f2-1ce142264dcd.png)

Do either of these approaches make sense for adding in extra docs links? Or, do we need to add a new design element, like a callout box or one of these?
![helptext box](https://www.smashingmagazine.com/wp-content/uploads/2011/11/skype1.jpg)

I think personally I'd try italic text and see if that works; you are welcome to experiment with other ideas too.